my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
```

Comment:
`os` and `glob` are imported but never used, so they could be removed. 
Also, `pyplot` is not imported everywhere, but that's a simple fix: `import matplotlib.pyplot as pyplot`